code/game/gamemodes/objective.dm

```diff
@@ -387,11 +387,12 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 	for(var/datum/mind/M in owners)
 		if(!M)
 			continue
-		var/datum/antagonist/changeling/changeling = M.has_antag_datum(/datum/antagonist/changeling)
-		if(!changeling)
+		if(!M.has_antag_datum(/datum/antagonist/changeling))
+			continue
+		if(isIPC(M.current))
```

Comment:
Shouldn't this be T.current not M.current?

Reply:
M checks for types of target
T checks to make sure wasn't a null selection
as is, this means it will always pick a changeling that is not an IPC, then make sure (T) is not null

Reply:
i'm confused as to why you want this to ensure it's a changeling

Reply:
Because that's how it was before your changes?

Reply:
I am still pretty sure we should be checking if the ipc is the TARGET (T) not the OWNER (M)

Reply:
i see what you're saying. it doesn't look like it reads as two different perspectives, only the target